ui/src/table.test.tsx

```diff
@@ -208,13 +208,37 @@ describe('Table.tsx', () => {
   })
 
   describe('sort', () => {
+    it('Does not render sort arrow on sortable columns by default', () => {
+      const { container } = render(<XTable model={tableProps} />)
+
+      // the sortingIcon class is present even if sort icon is not rendered, therefore we check for its child presence
+      expect(container.querySelector('i[class*=sortingIcon]')!.firstChild).not.toBeTruthy()
```

Comment:
Can we target the firstChild directly? Role/class would be the best, but if not we can also go with `'i[class*=sortingIcon]:first-child'`

```suggestion
      expect(container.querySelector('i[class*=sortingIcon]')!.firstChild).not.toBeInTheDocument()
```

Reply:
Not resolved yet, I still see a bunch of `!.firstChild.toBeTruthy`